### DISCUSSION TITLE
The appsettings.json on the config page is not working?

### DISCUSSION THREAD

**Discussion by Babolim** [vpnhood/VpnHood#276] [Q&A] [answered]
Hi, great VPN software.. 

However, if I copy the (updated) appsettings.json contents according to server configuration page, the server does not start (keeps trying to restart). Without the file it is working but I want to set "ExcludeLocalNetwork" to false and change the port.. Is there a bug in there?

The server keeps restarting if I just copy the contents that are displayed on the server config page. So by default these contents do not work? Of course I tried to change things myself.. But to no avail...

Thanks in advance for your help.

**Comment by Babolim** [comment]
Sorry, forget to mention this is on Linux..

**Reply by Babolim** [reply]
It seems it is only to the "NetFilter" part. If I add it, it fails to start..  It looks like this:
{
"RestAccessServer": null,
  "FileAccessServer": {
    "TcpEndPoints": [ "0.0.0.0:443", "[::]:443" ],
    "SslCertificatesPassword": null,
    "Tracking": {
      "TrackClientIp": false,
      "TrackLocalPort": false,
      "TrackDestinationIp": false,
      "TrackDestinationPort": false
  },
      "NetFilter": {
      "ExcludeLocalNetwork": false,
   }
  },
  "IsDiagnoseMode": false
}

If i remove the NetFilter part.. it runs fine.

Any suggestions?

Thanks.

**Comment by bobvhood** [comment, marked as answer]
Hi,

1) First, you have an additional comma in the json; please verify it with a JSON formatter
2) Please recheck the document: https://github.com/vpnhood/VpnHood/wiki/VpnHood-Server-Configuration
3) Please make sure you are using our later pre-release version (355)

Let me know if it doesn't solve the issue.

![image](https://user-images.githubusercontent.com/92276967/220071655-90a5f432-9758-4282-939e-e58b6e55457f.png)

**Reply by Babolim** [reply]
Thanks for your reply.

It seems the comma was the problem. As it is running with this config file:

{
  "RestAccessServer": null,
  "FileAccessServer": {
    "TcpEndPoints": [ "0.0.0.0:444", "[::]:444" ],
    "SslCertificatesPassword": null,
    "Tracking": {
      "TrackClientIp": false,
      "TrackLocalPort": false,
      "TrackDestinationIp": false,
      "TrackDestinationPort": false
  },
      "NetFilter": {
      "ExcludeLocalNetwork": false
   }
  },
  "IsDiagnoseMode": false
}

However, I can't ping or reach my local network, I thought the "ExcludeLocalNetwork": false would enable me to reach my local network ip addresses at home.. Any suggestions on how to accomplish that?

Thanks for any suggestions on that one. (I am still running the 350 version, would the 355 help or do I need to do something else?)

**Comment by trudyhood** [comment]
1) Please try the latest pre-release.
2) Make sure you uncheck the "Exclude LAN" option on the client too. By default client also doesn't send its LAN traffic to VpnHood servers
![image](https://user-images.githubusercontent.com/72891124/220491295-630e17a4-9e68-4c8e-aca5-7b9316efe1b1.png)

Let me know if it doesn't solve the issue.

**Reply by Babolim** [reply]
Thanks very much! I missed that setting..  Now all is working fine.. : )
